zerver/lib/streams.py

```diff
@@ -993,6 +995,45 @@ def can_access_stream_metadata_user_ids(stream: Stream) -> set[int]:
         )
```

Comment:
Would it have identical performance to have this function become `bulk_can_access_stream_metadata_user_ids([stream_id])`? It can be nice to not have duplicate implementations of security logic.